It seems that this feature was stabilized in `1.62` maybe we can bump the MSRV to that or even later version?

> error[E0658]: deriving `Default` on enums is experimental
>  --> /home/runner/.cargo/registry/src/github.com-1ecc6299db9ec823/defmt-parser-0.3.3/src/types.rs:3:24
>   |
> 3 | #[derive(Clone, Debug, Default, Eq, PartialEq)]
>   |                        ^^^^^^^
>   |
>   = note: see issue #86985 <https://github.com/rust-lang/rust/issues/86985> for more information
>   = note: this error originates in the derive macro `Default` (in Nightly builds, run with -Z macro-backtrace for more info)
>

--- a/.github/workflows/rust.yml
+++ b/.github/workflows/rust.yml
@@ -38,7 +38,7 @@ jobs:
           - stable
           - beta
           - nightly
-          - "1.60" # MSRV
+          - "1.65" # MSRV
 
     steps:
       - name: Checkout Sources

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -17,8 +17,8 @@ keywords = [
 license-file = "LICENSE"
 readme = "README.md"
 
-# `dep:...` in features is only available from 1.60 on
-rust-version = "1.60"
+# linux-embedded-hal requires 1.65 because of nix
+rust-version = "1.65"
 
 [dependencies]
 byteorder = { version = "1", default-features = false }

